src/main/resources/project.properties

```diff
@@ -0,0 +1,4 @@
+jfixtures.cmd.name=${project.name}
+jfixtures.cmd.version=${project.version}
+jfixtures.cmd.link=For more info please visit https://github.com/vkorobkov/jfixtures-cmd
```

Comment:
I thought we can broadcast pure url `https://github.com/vkorobkov/jfixtures-cmd` from pom.xml `scm` section?